src/components/utils/http.js

```diff
@@ -22,7 +22,8 @@ const errorMessage = (status) => {
   }
 }
 
-const handleError = (status, callback) => {
+const handleError = ({status, response}, callback) => {
+  console.error(status, response)
   const message = errorMessage(status)
   Tracker.sendError(message)
```

Comment:
Oh, I can see it now. It can be related to the referer or the token. Cool, I will add the response body to the Sentry error then. Although for other 4xx errors other than 401 the response body will be empty unless handled on the server. I assume status code will be enough in that case